### ISSUE TITLE
Sidebar not redrawn after S/MIME certificate chain verification screen, not cleared on exit

### ISSUE THREAD

**Issue by nabijaczleweli** [neomutt/neomutt#3339]
## Expected Behaviour

Not that.

## Actual Behaviour

![image](https://user-images.githubusercontent.com/6709544/162030615-d099f712-770a-4c28-b5ab-2778cfa2b04d.png)
![image](https://user-images.githubusercontent.com/6709544/162031267-765aef42-4f82-4c66-9ac0-50a4b5714821.png)

## Steps to Reproduce

1. Open an S/MIME-signed mail
2. Confirm CA
3. Confirm certificate
4. ruh-roh

## How often does this happen?

- Always

Well, once. I'm not sure how to clear the certificate confirmation cache to re-try, and this is my bank statement, so.

## NeoMutt Version

```
NeoMutt 20211029-519-ac10a7
Copyright (C) 1996-2020 Michael R. Elkins and others.
NeoMutt comes with ABSOLUTELY NO WARRANTY; for details type 'neomutt -vv'.
NeoMutt is free software, and you are welcome to redistribute it
under certain conditions; type 'neomutt -vv' for details.

System: Linux 5.10.0-13-amd64 (x86_64)
ncurses: ncurses 6.2.20201114 (compiled with 6.2.20201114)
libidn: 1.33 (compiled with 1.33)
GPGME: 1.14.0-unknown
GnuTLS: 3.7.1
libnotmuch: 5.3.0
storage: tokyocabinet, lmdb
compression: zlib, zstd

Configure options: CC=cc LD=ld.lld --asan --mandir=/usr/share/man --libexecdir=/usr/libexec --with-mailpath=/var/mail --gpgme --lua --notmuch --gnutls --gss --idn --mixmaster --sasl --tokyocabinet --lmdb --zlib --zstd --testing --asan

Compilation CFLAGS: -g -O2 -std=c99 -D_ALL_SOURCE=1 -D_GNU_SOURCE=1 -D__EXTENSIONS__ -I/usr/include -I/usr/include/lua5.4 -DNCURSES_WIDECHAR -isystem /usr/include/mit-krb5 -fsanitize=address -O0 -fno-omit-frame-pointer -fno-common

Default options:
  +attach_headers_color +compose_to_sender +compress +cond_date +debug 
  +encrypt_to_self +forgotten_attachments +forwref +ifdef +imap +index_color 
  +initials +limit_current_thread +multiple_fcc +nested_if +new_mail +nntp +pop 
  +progress +quasi_delete +regcomp +reply_with_xorig +sensible_browser +sidebar 
  +skip_quoted +smtp +status_color +timeout +tls_sni +trash 

Compile options:
  -autocrypt +fcntl -flock -fmemopen +futimens +getaddrinfo +gnutls +gpgme +gss 
  +hcache -homespool +idn +inotify -locales_hack +lua +mixmaster +nls +notmuch 
  -openssl +pgp +regex +sasl +smime -sqlite +sun_attachment 

Devel options:
  asan 

MAILPATH="/var/mail"
MIXMASTER="mixmaster"
PKGDATADIR="/usr/share/neomutt"
SENDMAIL="/usr/sbin/sendmail"
SYSCONFDIR="/etc"

To learn more about NeoMutt, visit: https://neomutt.org
If you find a bug in NeoMutt, please raise an issue at:
    https://github.com/neomutt/neomutt/issues
or send an email to: <neomutt-devel@neomutt.org>
```

## Extra Info

* Operating System and its version: Bullseye
* Were you using multiple copies of NeoMutt at once? No
* Were you using 'screen' or 'tmux'? no
* Is your email local (maildir) or remote (IMAP)? IMAP

**Comment by flatcap** [comment]
Well, @nabijaczleweli, you **do** come up with the best bugs :-)

Thanks, I'll have think and a play around later.

**Comment by flatcap** [comment]
I've pushed a possible fix (5ede6666f) to branch [[devel/refresh]](https://github.com/neomutt/neomutt/tree/devel/refresh)

Opening the email is causing an external program to be called, to verify the certificate.
I've put a refresh into the Pager when it starts, to make sure everything gets repainted.

That should be enough; I'll have another think in the morning.

**Comment by flatcap** [comment]
Fixed in d5e77868f.
